.github/workflows/release.yml

```diff
@@ -0,0 +1,186 @@
+          draft: true
+          files: |
+            vscode-xml-*-${{ env.XML_SERVER_VERSION }}-${{ github.run_number }}.vsix
+            ../staging/zips/lemminx-*
+            ../staging/checksums/lemminx-*
+  release-job:
+    environment: 'publishing'
+    runs-on: ubuntu-latest
+    needs: packaging-job
+    steps:
+      - name: Check Out VS Code XML
+        uses: actions/checkout@v2
+      - name: Set Up NodeJS
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Install dependencies
+        run: npm install -g typescript "vsce" "ovsx"
+      - run: echo "XML_SERVER_VERSION=$(cat package.json | jq -r .xmlServer.version)" >> $GITHUB_ENV
+      - name: Set the link to download the binary server
+        env:
+            downloadLocation: https://github.com/redhat-developer/vscode-xml
+        if: ${{ inputs.publishToMarketPlace == 'true' }}
+        run: sed -i -e "s|${{ env.downloadLocation }}/releases/download/latest|${{ env.downloadLocation }}/releases/download/${{ env.XML_SERVER_VERSION }}|g" package.json
+      - name: Download VSIX & LemMinX Server Uber Jar
+        uses: actions/download-artifact@v3
+      - name: Build vscode-xml
+        run: |
+          npm install
+          npm run vscode:prepublish
+      - run: |
+          mkdir server/
+          mv lemminx-uber-jar/org.eclipse.lemminx*-uber.jar server/
+      - name: Prepare Pre-Release
+        if: ${{ github.event_name == 'schedule' || inputs.publishPreRelease == 'true' }}
+        run: |
+          npx gulp prepare_pre_release
+          echo "EXT_VERSION=$(cat package.json | jq -r .version)" >> $GITHUB_ENV
+          echo "publishPreReleaseFlag=--pre-release" >> $GITHUB_ENV
+      - name: Publish to VS Code Marketplace
+        if: ${{ github.event_name == 'schedule' || inputs.publishToMarketPlace == 'true' || inputs.publishPreRelease == 'true' }}
+        run: |
+          for platformVsix in vscode-xml/vscode-xml-*-${{ env.XML_SERVER_VERSION }}-${GITHUB_RUN_NUMBER}.vsix; do
+            echo vsce publish -p ${{ secrets.VSCODE_JAVA_MARKETPLACE }} --packagePath ${platformVsix}
+          done
+          echo vsce publish -p ${{ secrets.VSCODE_JAVA_MARKETPLACE }} --target win32-ia32 win32-arm64 linux-arm64 linux-armhf alpine-x64 alpine-arm64 ${{ env.publishPreReleaseFlag }}
+      - name: Publish to OpenVSX Registry
+        if: ${{ github.event_name == 'schedule' || inputs.publishToOVSX == 'true' || inputs.publishPreRelease == 'true' }}
+        run: |
+          for platformVsix in vscode-xml/vscode-xml-*-${{ env.XML_SERVER_VERSION }}-${GITHUB_RUN_NUMBER}.vsix; do
+            echo ovsx publish -p ${{ secrets.OPEN_VSX_ACCESS_TOKEN }} --packagePath ${platformVsix}
+          done
+          echo ovsx publish -p ${{ secrets.OPEN_VSX_ACCESS_TOKEN }} --target win32-ia32 win32-arm64 linux-arm64 linux-armhf alpine-x64 alpine-arm64 ${{ env.publishPreReleaseFlag }}
```

Comment:
should publish generic vsix. See https://github.com/redhat-developer/vscode-xml/issues/865